pkgs/development/libraries/blkhash/default.nix

```diff
@@ -0,0 +1,66 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchFromGitLab
+, makeWrapper
+, meson
+, ninja
+, pkg-config
+, cmake
+, openssl
+, enableLibnbd ? stdenv.isLinux, libnbd
+, qemu-utils
+, asciidoc
+, python3Packages
+}:
+let
+  unity = fetchFromGitHub {
+    owner = "ThrowTheSwitch";
+    repo = "Unity";
+    rev = "5204c1bacf37e0c38211d2fbb6d9796e410223f8";
+    sha256 = "i4uQOG/Q86AaQINJoC/65QA/ZEUOXzaVceR2inw32yU=";
+  };
+in
+stdenv.mkDerivation rec {
+  version = "0.6.1";
+  pname = "blkhash";
+
+  src = fetchFromGitLab {
+    owner = "nirs";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "+7SIZdvkdMYVgeklbqU0MHKc1VJnk9opqhsCmkg6bcI=";
+  };
+
+  postUnpack = ''(
+    cd "$sourceRoot/subprojects"
+    cp -R --no-preserve=mode,ownership ${unity} unity
+  )'';
+
+  dontUseCmakeConfigure = true;
+
+  buildInputs = [
+    makeWrapper
+    meson
+    ninja
+    pkg-config
+    cmake
+    openssl
+    qemu-utils
+    asciidoc
```

Comment:
Sorting this list would be nice.

Reply:
I was struggling with deciding what order should be used here. Many derivations in nixpkgs don't use lexicographical order. I would suggest to keep `buildInputs` sorted, but for attribute set on top of nix file I suggest to place all things related to nix tooling (like lib or fetchFromGitLab) on top of other arguments.

Reply:
I think sorting helps with simple lists. The order of top level things
should depend on the semantics. Your suggestion sounds good.